packages/radix-vue/src/shared/useBodyScrollLock.ts

```diff
@@ -7,93 +7,98 @@ import { nextTick, onBeforeUnmount, ref, watch } from 'vue'
 import { defu } from 'defu'
 import { injectConfigProviderContext } from '@/ConfigProvider/ConfigProvider.vue'
 
-const useInitialOverflowStyle = createGlobalState(() => ref<string | undefined>())
-const useBodyLockStackCount = createGlobalState(() => ref(0))
+const useBodyLockStackCount = createGlobalState(() => {
```

Comment:
one reason we save the initial overflow style is because we wanna preserve the user defined overflow styling. 

eg:
user set "overflow: hidden", if we didnt preserve this after unlocking body lock, it would revmoe the styling, removing the default behavior

Reply:
Oh make sense. My bad I'll put it back.